recipe/meta.yaml

```
@@ -19,31 +19,40 @@ requirements:
   host:
     - python >=3.5
     - pip
+    - hatchling >=1.12.2
+    - hatch-vcs >=0.2.0
+    - wheel
   run:
     - python >=3.5
-    - six
-    - cryptography
+    - cryptography >=41
 
 test:
+  imports:
+    - conda_content_trust
```

```suggestion
    - conda_content_trust
    - conda_content_trust.signing
    - conda_content_trust.root_signing
```

Since `__init__` only imports `__version__`, let's add a couple more just to check things are indeed in place.